db/migrations/20210513002725-update-container-weight-unit.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return await queryInterface.renameColumn(
+      'LineItems',
+      'containerWeightGrams', // Old name
+      'containerWeightKilos', // New name
+    )
+  },
+
+  down: async (queryInterface, Sequelize) => {
+    return await queryInterface.renameColumn(
+      'LineItems',
+      'containerWeightKilos',
+      'containerWeightGrams',
+    )
+  },
+}
```

Comment:
Question (blocking): will we ever want more precision than 1 kilo? e.g. 1.5 kilos? If so we should leave this as is. The column is an integer type to avoid floating point semantics.

Reply:
Yeah I was on the edge about that 🤔 I'll try an alternative.

Reply:
@bion I reverted the backend change and handled the weight conversion on the frontend.